tests/conftest.py

```diff
@@ -61,11 +62,57 @@ def db_fixtures(db_tables):
     team = Team(name='TeamName', organization=organization)
     user = User(first_name='User', last_name='Name', email='user@name.test')
     user.set_password('hunter2')
+    user2 = User(first_name='User2', last_name='Name2', email='user2@name.test')
+    user2.set_password('hunter2')
     project = Project(name='ProjectName')
+
+    # user1 consents
+    consent1_1 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.accepted.name,
+                         user=user)
+    consent1_2 = Consent(type=ConsentType.gdpr.name,
+                         category='newsletter',
+                         status=ConsentStatus.rejected.name,
+                         user=user)
+    consent1_3 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.accepted.name,
+                         user=user)
+    consent1_4 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.rejected.name,
+                         user=user)
+    consent1_5 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.accepted.name,
+                         user=user)
+
+    # user2 consents
+    consent2_1 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.preferences.name,
+                         status=ConsentStatus.accepted.name,
+                         user=user2)
+    consent2_2 = Consent(type=ConsentType.gdpr.name,
+                         category='newsletter',
+                         status=ConsentStatus.rejected.name,
+                         user=user2)
+    consent2_3 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.accepted.name,
+                         user=user2)
+    consent2_4 = Consent(type=ConsentType.cookie.name,
+                         category=CookieConsentCategory.statistics.name,
+                         status=ConsentStatus.rejected.name,
+                         user=user2)
```

Comment:
It's to ensure that the consents that are retrieved for User1 are User1's latest consents, and not just any latest consents, as User2's consents have more recent timestamp.

Reply:
- Ok, that makes sense, it just feels like a very implict relationship. I don't think it's easy to tell that it is being tested for, just by looking at the tests. I don't have a better suggestion though.
- The `models` fixture should then make all new objects available to test cases like the other fixtures, instead of just selecting the first user.
- Is it possible to reduce the number of fixtures? It doesn't seem strictly necessary to cover all those parameters. At least for the use case you mentioned only one should be enough.
- I think the remaining fixtures should be named semantically, not with random numbers. For example `consent_gdpr_rejected` instead of `consent1_2`.

Reply:
Yeah, it's true that not all of them are necessary and most of them were given just to test different `test`/`category`/`status` values, but the tests should take care of that.
- Reducing and renaming of fixtures addressed in https://github.com/DD-DeCaF/iam/commit/e0b711de0946b19c17c77134c8505b432fff50eb
- Making new objects available addressed in https://github.com/DD-DeCaF/iam/commit/a9e3b23bd4e86b179a0242e72ec582d07c7fa307 and https://github.com/DD-DeCaF/iam/commit/ee918a77947491471b84710dc893b65ea5f4c405